protocol/messenger_sync_wallets_test.go

```diff
@@ -146,12 +146,12 @@ func (s *MessengerSyncWalletSuite) TestSyncWallets() {
 	woAccounts := accounts.GetWatchOnlyAccountsForTest()
 	err = s.m.settings.SaveOrUpdateAccounts(woAccounts, false)
 	s.Require().NoError(err)
-	dbWoAccounts1, err := s.m.settings.GetWatchOnlyAccounts()
+	dbWoAccounts1, err := s.m.settings.GetWatchOnlyAccounts(false)
```

Comment:
Don't we want to follow the same approach with this one?

Reply:
Not sure how I missed it, was in a rush, anyhow here is a PR handling this function as well, https://github.com/status-im/status-go/pull/3807. Thanks.